src/react-components/src/components/02-molecules/Cards/EditionCard.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from "react";
+import bem from "../../../utils/bem";
+
+import EditionCard from "./EditionCard";
+import { action } from "@storybook/addon-actions";
+
+export default {
+  title: "Edition Card",
+  component: EditionCard,
+};
+
+export const editionCardWithFullData = () => <EditionCard
+  id="card#1"
+  blockName=""
+  coverUrl="https://placeimg.com/300/400/arch"
+  editionHeadingElement={<a href="edition-link" className={bem("link", [], "heading")} >2004 Edition</a>}
+  editionInfo={["Published in New York by Random House", "Written in English", "Under Creative Commons License"]}
+  readOnlineLink="#readOnlineUrl"
+  downloadLink="#downloadUrl">
+</EditionCard>;
+
+export const editionCardWithOneLinks = () => <EditionCard
+  id="card#1"
+  blockName=""
+  coverUrl="https://placeimg.com/300/400/arch"
+  editionHeadingElement={<a href="edition-link" className={bem("link", [], "heading")} >2004 Edition</a>}
+  editionInfo={["Published in New York by Random House", "Written in English", "Under Creative Commons License"]}
+  readOnlineLink="#readOnlineUrl">
+</EditionCard>;
+
+export const editionCardMissingLinks = () => <EditionCard
+  id="card#1"
+  blockName=""
+  coverUrl="https://placeimg.com/300/400/arch"
+  editionHeadingElement={<a href="edition-link" className={bem("link", [], "heading")} >2004 Edition</a>}
+  editionInfo={["Published in New York by Random House", "Written in English", "Under Creative Commons License"]}
+  >
+</EditionCard>;
+
+export const editionCardWithSomeEditionInfo = () => <EditionCard
+  id="card#1"
+  blockName=""
+  coverUrl="https://placeimg.com/300/400/arch"
+  editionHeadingElement={<a href="edition-link" className={bem("link", [], "heading")} >2004 Edition</a>}
+  editionInfo={["Published in New York", "Under Creative Commons License"]}
+  readOnlineLink="#readOnlineUrl"
+  downloadLink="#downloadUrl">
+</EditionCard>;
+
+export const editionCardWithEmptyEditionInfo = () => <EditionCard
+  id="card#1"
+  blockName=""
+  coverUrl="https://placeimg.com/300/400/arch"
+  editionHeadingElement={<a href="edition-link" className={bem("link", [], "heading")} >2004 Edition</a>}
+  editionInfo={[]}
```

Comment:
I think it's annoying, as a developer using a library, to have to pass in empty values if the prop is required. On a more realistic use-case, if you are getting data from a server, it's likely that instead of returning an empty array (or empty object), you can get `undefined` as a value. In that case, this will break. It's possible that NYPL's API will return empty values if there is no data, but that may not be the case for anyone else. I think a default prop of `[]` may work better.

Reply:
Agreed!  I'm gonna go in and add defaultProps to stuff in this PR, but I would love to go back and add them to everything - especially all the `baseClass`, `modifiers`, etc.  (They're just new to me (thank you!) and I think they're super nifty)

Reply:
Satisfying future work for post-MVP